enclave_manager/avalon_enclave_manager/work_order_kv_delegate.py

```diff
@@ -51,9 +52,26 @@ def cleanup_work_orders(self, wo_ids=None):
             False - Otherwise
 
         """
-        # @TODO : Enable cleanup for wo_ids passed in argument.
-        # As of now a blanket cleanup is being done.
-        processing_list = self._kv_helper.lookup("wo-processing")
+        # Lookup workers that belong to this pool i.e.- sharing same worker_id.
+        # In case of Singleton, this will be a single worker_id whereas in case
+        # of worker pool mode, it will be a list of worker's identity (the
+        # enclave_id of a WPE).
+        workers_in_pool = self._kv_helper.get("worker-pool", self._worker_id)
+        if workers_in_pool is not None:
+            workers = workers_in_pool.split(",")
+        else:
+            logger.info("No worker found in pool. Cleanup not required")
+            return True
+        processing_list = []
+        for worker in workers:
+            wo_id_active = self._kv_helper.get("wo-worker-active", worker)
```

Comment:
We are creating so many tables, can you add a comment in code / document of each 
 table name and what is structure(key and value format). It is difficult to follow the code otherwise.